src/core/spdnsclient/net.go

```diff
@@ -0,0 +1,57 @@
+package spdnsclient
```

Comment:
Same comment as in `customdial`

Reply:
It's just copied from golang source, but without reading `/etc/resolv.conf` and Unix things.